### ISSUE TITLE
FPWD Request for W3C Accessibility Guidelines (WCAG) 3.0

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/transitions#303]
# Document title, URLs, estimated publication date
Title: W3C Accessibility Guidelines (WCAG) 3.0
Anticipated Publication URI: https://www.w3.org/TR/wcag-3.0/
Editors' Draft URI: https://w3c.github.io/silver/guidelines/
Desired Publication Date: 2021-01-19

# Abstract
The W3C Accessibility Guidelines (WCAG) 3.0 provide a wide range of recommendations for making web content more accessible to users with disabilities. Following these guidelines will address many of the needs of users with blindness, low vision and other vision impairments; deafness and hearing loss; limited movement and dexterity; speech disabilities; sensory disorders; cognitive and learning disabilities; and combinations of these. These guidelines address accessibility of web content on desktops, laptops, tablets, mobile devices, wearable devices, and other web of things devices. They address various types of web content including static content, interactive content, visual and auditory media, and virtual and augmented reality. The guidelines also address related web tools such as user agents (browsers and assistive technologies), content management systems, authoring tools, and testing tools.

Each guideline in this standard provides information on accessibility practices that address documented user needs of people with disabilities. Guidelines are supported by multiple outcomes to determine whether the need has been met. Guidelines are also supported by technology-specific methods to meet each outcome.

This specification is expected to be updated regularly to keep pace with changing technology by updating and adding methods, outcomes, and guidelines to address new needs as technologies evolve. For entities that make formal claims of conformance to these guidelines, several levels of conformance are available to address the diverse nature of digital content and the type of testing that is performed.

W3C Accessibility Guidelines 3.0 is a successor to Web Content Accessibility Guidelines 2.2 [WCAG22] and previous versions, but does not deprecate these versions. WCAG 3.0 will incorporate content from and partially extend User Agent Accessibility Guidelines 2.0 [UAAG20] and Authoring Tool Accessibility Guidelines 2.0 [ATAG20]. While there is a lot of overlap between WCAG 2.X and WCAG 3.0, WCAG 3.0 includes additional tests and different scoring mechanisms. As a result, WCAG 3.0 is not backwards compatible with WCAG 2.X. WCAG 3.0 does not supersede WCAG 2.2 and previous versions; rather, it is an alternative set of guidelines. Once these guidelines become a W3C Recommendation, the W3C will advise developers, content creators and policy makers to use WCAG 3.0 in order to maximize future applicability of accessibility efforts. However, content that conforms to earlier versions of WCAG continue to conform to those versions.

See WCAG 3 Introduction for an introduction and links to WCAG technical and educational material.

# Status
This section describes the status of this document at the time of its publication. Other documents may supersede this document. A list of current W3C publications and the latest revision of this technical report can be found in the W3C technical reports index at https://www.w3.org/TR/.

This is a First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 by the Accessibility Guidelines Working Group, with primary development in the Silver Task Force and Silver Community Group. WCAG 3 is intended to be easier to understand and more flexible than WCAG 2. The flexibility is to address different types of web and digital content, apps, and tools — as well as a broader range of people with disabilities. WCAG 3 proposes a different name, scope, structure, and conformance model from WCAG 2.

This publication is the result of several years of research and development. It realizes many of the goals arising from that work and articulated in the Requirements for WCAG 3. This First Public Working Draft is a very early draft, published to obtain review of the direction proposed. The Working Group seeks input on the following general questions:

*    How well will this proposal of conformance work in your type of organization and why?
*    What do you see as the possible tradeoffs in using a flexible conformance approach? Would you want to see multiple conformance models?
*    How will the change to a different conformance model than WCAG 2 affect your organization? How readily would you see moving from WCAG 2 to WCAG 3 when it is complete?
*    Are there usability improvements that would make WCAG 3 easier to use and find information? Is the structuring of the How-To, outcomes, and methods clear? Are there improvements to structure and style that you would like to see? Is there another way we can make content more usable?

Reveiwers are encouraged to consider the more extensive review questions in the blog post "WCAG 3 FPWD Published".

To comment, file an issue in the W3C silver GitHub repository. The Working Group requests that public comments be filed as new issues, one issue per discrete comment. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to public-agwg-comments@w3.org (comment archive). The Working Group requests comments on this draft be sent by 26 February 2021. In-progress updates to the guidelines can be viewed in the public editors' draft.

This document was published by the Accessibility Guidelines Working Group as a First Public Working Draft. This document is intended to become a W3C Recommendation.

GitHub Issues are preferred for discussion of this specification.

Publication as a First Public Working Draft does not imply endorsement by the W3C Membership.

This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.

This document was produced by a group operating under the 1 August 2017 W3C Patent Policy. W3C maintains a public list of any patent disclosures made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains Essential Claim(s) must disclose the information in accordance with section 6 of the W3C Patent Policy.

This document is governed by the 15 September 2020 W3C Process Document. 

# Is it a delta specification intended to become a W3C Recommendation?
No

# Link to group's decision to request transition
https://lists.w3.org/Archives/Public/w3c-wai-gl/2021JanMar/0007.html

# Information about implementations known to the Working Group
None yet

# Formal Objections

The Call for Consensus to publish had 5 objections. One was [addressed and withdrawn](https://lists.w3.org/Archives/Public/w3c-wai-gl/2020OctDec/0103.html). Editorial changes were made to [address the others](https://lists.w3.org/Archives/Public/w3c-wai-gl/2020OctDec/0190.html), but the objectors did not withdraw the objections and requested they be treated as Formal Objections. Due to otherwise broad support to publish and in accordance with the [AG WG Decision Policy](https://www.w3.org/WAI/GL/decision-policy), the chairs declared the CfC as "passed with objections", understanding that Director review is still needed.

**Comment by jennyliang220** [comment]
Hi @plehegar @michael-n-cooper , just let you know, WCAG 3.0 is using different [shortname strategy](https://github.com/w3c/manual-of-style/pull/13#issuecomment-756587936) from WCAG 2.x.

> Anticipated Publication URI: https://www.w3.org/TR/wcag-3.0/

For WCAG2.1, it's: https://www.w3.org/TR/WCAG21/

**Comment by slhenry** [comment]
I noticed that shortname format change, and don't like it. I don't know why it was chosen.

**Comment by brewerj** [comment]
@plehegar and @michael-n-cooper please note that we cannot switch our short-name strategy without prior agreement as there is substantial background on the existing approach, and this will trigger problems in multiple resources, as well as discoverability problems. 
This should be wcag30 -- any reason why not using that, and can we confirm continuing as expected? 

e.g. if this is W3C's new approach, Michael you're feeling required to follow, please advise -- but this was a surprise for the messaging team.

**Comment by plehegar** [comment]
There were 4 formal objections raised against this transition.

Regarding attribution of contributions, contributions from individuals should
be acknowledged at an appropriate level in the document. As decided last week, the Group
is starting an effort to enhance and clarify its acknowledgments
section. A Note in Section D.1 of the document was added to indicate
this. My expectation is for the Working Group to revise the
acknowledgments section as needed and consider republishing a new
Working Draft within 6 months of the publication of the First Public
Wording Draft. As such, this formal objection remains and will need to
be revisited within 6 months of the first publication unless it is
withdrawn.

Regarding the request for clarification for the specific role of
editors, the Chairs already reminded the objector of the requirements of
the W3C Process. Any expectation or requirement beyond what is
required by the W3C Process may be documented by the Working Group
chairs, at their discretion, and they have done so. Given that this formal objection does not
cite technical arguments nor propose changes to the document, this
formal objection is overruled.

Regarding the barriers to participation, it is my understanding that the Accessibility Guidelines Working Group Chairs and Task Force Facilitators have been oriented to the updated W3C Code of Ethic and Professional Conduct (CEPC), which provides more detailed guidance on appropriate and inappropriate interactions in the Working Group and Task Forces, and that AGWG Chairs are more actively monitoring TF discussions for any potential CEPC concerns. The Process Community group is also considering adding tooling requirements to the W3C
Process[2]. All Group participants, including Chairs and Editors, must abide
by the terms and spirit of the W3C Code of Ethics and
Professional Conduct. Any expectation or requirement beyond what is
required by the W3C Process may be documented by the Working Group
chairs, at their discretion. Issues related to CEPC and individual participation should
be resolved through the designated ombudsperson.
 Some barriers, like required expertise and time commitment, may remain
due to the nature of the work and the high volume of information. W3C
has been improving over the years to remove barriers, to be as inclusive as
possible, and will continue to do so. A note was added in the document
to invite further feedback. This formal objection is overruled and
specific issues may be raised in the Working Group or the appropriate
related groups (Positive Work Environment Community Group, Revising
W3C Process Community Group).

Regarding the conformance model, while the content of a Working Draft
may always be misinterpreted by readers, a Working Draft may be
published even if its content is considered unstable and does not meet
all Working Group requirements. Publishing a Working Draft is a signal for
the community to provide feedback. Section 8 of the document indicates
clearly that the Working Group is seeking feedback on the conformance
approach as a whole. Given that this formal objection does not cite
technical arguments nor propose changes to the document, this formal
objection is overruled. Specific issues may be raised against the
conformance model approach in the future.

In conclusion, on behalf of the W3C Director, this transition request is
approved. The formal objection related to attributions will be revisited
within 6 months unless it is withdrawn. The formal objections on
editors, barriers to participation, and the conformance model are
overruled. If new information arises, new issues may be raised in
the Working Group.



[1] https://www.w3.org/Consortium/cepc/cepc-20200716/
[2] https://github.com/w3c/w3process/issues/435
[3] https://www.w3.org/Consortium/cepc/cepc-20200716/#dfn-ombudsperson

**Comment by plehegar** [comment]
https://www.w3.org/TR/2021/WD-wcag-3.0-20210121/

**Comment by PressureTechs** [comment]
https://www.w3.org/TR/2021/WD-wcag-3.0-20210121/
